### PR TITLE
fix label not int

### PR DESCRIPTION
**Pull request by wonderflow**

### CODE REVIEW

File: pkg/appfile/service.go [hunk 1]
```diff
@@ -97,12 +99,16 @@ func (s Service) RenderService(tm template.Manager, name, ns, image string) (
...
-	traits := []v1alpha2.ComponentTrait{}
-	for k, v := range traitKeys {
-		ts, err := evalTraits(tm.LoadTemplate(k), ctxData, intifyValues(v))
+	traits := make([]v1alpha2.ComponentTrait, 0)
+	for traitType, traitData := range traitKeys {
+		ts, err := evalTraits(tm.LoadTemplate(traitType), ctxData, intifyValues(traitData))
 		if err != nil {
 			return nil, nil, fmt.Errorf("eval traits failed: %w", err)
 		}
+		// one capability corresponds to one trait only
```
Comment (hongchaodeng) [inline pkg/appfile/service.go:108]: Adding the label should be enough to fix the problem.
I don't think we should add such constraint in the code right now. 
It is that we won't tell the story on the tutorials that we agree on.

File: pkg/appfile/service_test.go [hunk 2]
```diff
@@ -0,0 +1 @@
+package appfile
```
Comment (hongchaodeng) [inline pkg/appfile/service_test.go:1]: delete this empty file?

**Review by hongchaodeng — APPROVED** [review]
lgtm

File: pkg/appfile/service_test.go (hunk 2 above)
Comment (wonderflow) [reply pkg/appfile/service_test.go:1]: I will add tests in the following PRs here.

**Pull request merged**